include/beman/optional/optional.hpp

```diff
@@ -403,6 +409,13 @@ class optional {
         std::destroy_at(std::addressof(value_));
         engaged_ = false;
     }
+
```

Comment:
Reviewing for creating LWG issues:
I believe this, which is necessary for https://eel.is/c++draft/optional.monadic#note-1, is already covered by the current wording.